README.md

```diff
@@ -110,4 +110,30 @@ A script will run automatically, asking for your settings and desired configurat
 
 <hr>
 
-**MeiliSearch** provides and maintains many **SDKs and Integration tools** like this one. We want to provide everyone with an **amazing search experience for any kind of project**. If you want to contribute, make suggestions, or just know what's going on right now, visit us in the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
\ No newline at end of file
+**MeiliSearch** provides and maintains many **SDKs and Integration tools** like this one. We want to provide everyone with an **amazing search experience for any kind of project**. If you want to contribute, make suggestions, or just know what's going on right now, visit us in the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
```

Comment:
For me, this paragraph was the footer. Should it be at the end of the README?

Reply:
Oh my bad, I move it to the bottom! 👍